api/src/pcapi/core/providers/repository.py

```diff
@@ -123,6 +123,24 @@ def get_cgr_cinema_details(cinema_id: str) -> models.CGRCinemaDetails:
     return cinema_details
 
 
+def get_ems_cinema_details(cinema_id: str) -> models.EMSCinemaDetails:
+    cinema_details = (
+        models.EMSCinemaDetails.query.join(models.CinemaProviderPivot)
+        .join(models.CinemaProviderPivot.provider)
+        .filter(models.CinemaProviderPivot.idAtProvider == cinema_id)
+        .filter(models.Provider.localClass == "EMSStocks")
+        .one()
+    )
+    return cinema_details
+
+
+def update_ems_cinema_sync_version(venue_provider: models.VenueProvider, version: int) -> None:
```

Comment:
```suggestion
def set_ems_cinema_sync_version(venue_provider: models.VenueProvider, version: int) -> None:
```
Afin de ne pas laisser penser que l'on fait directement un UPDATE commité.

Reply:
done @francois-seguin